tw-tkms-starter/src/main/java/com/transferwise/kafka/tkms/config/TkmsAutoConfiguration.java

```diff
@@ -51,4 +49,9 @@ public List<ITkmsMessageDecorator> messageDecorators() {
     return Collections.emptyList();
   }
 
+  @Bean
```

Comment:
Probably would be better to declare the autowire usage as
`@Autowired(required = false)`
Then you don't need to explicitly provide an empty list bean.

Reply:
Fixed that.